GliaWidgets/SecureConversations/ChatTranscript/SecureConversations.TranscriptModel+GVA.swift

```diff
@@ -0,0 +1,127 @@
+import Foundation
+
+private extension String {
+    static let gvaOptionUrlTargetModal = "modal"
+    static let gvaOptionUrlTargetSelf = "self"
+}
+
+extension SecureConversations.TranscriptModel {
+    func quickReplyOption(_ gvaOption: GvaOption) -> QuickReplyButtonCell.Props {
+        let action = Cmd { [weak self] in
+            self?.gvaOptionAction(for: gvaOption)()
+            self?.action?(.quickReplyPropsUpdated(.hidden))
+        }
+        return .init(
+            title: gvaOption.text,
+            action: action
+        )
+    }
+
+    func gvaOptionAction(for option: GvaOption) -> Cmd {
+        // If `option.destinationPdBroadcastEvent` is specified,
+        // this is broadcast event button, which is not supported
+        // on mobile. So an alert should be shown.
+        if option.destinationPdBroadcastEvent != nil {
+            return broadcastEventButtonAction()
+        }
+
+        // If option contains `url`, then it's URL Button
+        if let urlString = option.url, let url = URL(string: urlString) {
+            return urlButtonAction(url: url, urlTarget: option.urlTarget)
+        }
+
+        // Otherwise it's Postback Button and option should be sent
+        // to the server as `SingleChoiceOption`
+        return postbackButtonAction(for: option)
+    }
+
+    func postbackButtonAction(for option: GvaOption) -> Cmd {
```

Comment:
The URL button action is internal but the `urlButtonAction` is not. Is this one used somewhere else or it can also go to the private extension?

Reply:
My fault. Fixed b535b617199a51a3e7a3f0152d46279ade0dff9f